configs/jsactions/rollup.config.js

```diff
@@ -45,45 +45,47 @@ export default async args => {
                 i === 0 ? clear({ targets: [outDir] }) : null,
                 !isWeb
                     ? collectDependencies({
-                          onlyNative: false,
-                          outputDir: outDir,
-                          widgetName: fileOutput
-                      })
+                        onlyNative: false,
+                        outputDir: outDir,
+                        widgetName: fileOutput
+                    })
                     : null,
                 nodeResolvePlugin,
                 typescriptPlugin,
                 bigJsImportReplacer(),
-                i === files.length - 1
```

Comment:
Removing this line to me sounds like rollup will try to identify same dependencies for all files.. but the dependencies are based on the package.json

Reply:
I added it back in. As discussed, we will meet at 3 to conclude these changes